modules/adding-notebook-pod-tolerations.adoc

```diff
@@ -0,0 +1,41 @@
+:_module-type: PROCEDURE
+
+[id="adding-notebook-pod-tolerations_{context}"]
+= Adding notebook pod tolerations
+
+[role='_abstract']
+You can add a toleration to notebook pods to allow them to be scheduled on tainted nodes. This is useful if you want to dedicate specific worker nodes to running notebooks, as tainting them will prevent other workloads from running on them. 
```

Comment:
Provide a brief description of a toleration and taints (OpenShift concept) - see this doc for info:
https://docs.openshift.com/container-platform/4.14/nodes/scheduling/nodes-scheduler-taints-tolerations.html

You can also add this as a link under the "Additional resources" section

Also - avoid using future tense - for example "will prevent" > "prevents"